cdserver/README.md

```diff
@@ -22,3 +22,5 @@ Ruby on Rails 5 API
   * Run `rails cucumber` to run cucumber tests
   * Run `rails cucumber:html` to print report of cucumber tests to `reports/feature/results/index.html`
   * After running tests, code coverage report can be found at `reports/feature/coverage/index.html`
+  * Run `brakeman` for brakeman tests
```

Comment:
please add:
Run `brakeman` for brakeman security testing, including the OWASP top 10 issues e.g. XSS and SQL injection.
Assuming that is actually true :-)